src/netbase.cpp

```diff
@@ -651,7 +667,7 @@ void CNetAddr::SetRaw(Network network, const uint8_t *ip_in)
             memcpy(ip, ip_in, 16);
             break;
         default:
-            assert(!"invalid network");
+	    assert(true);  // invalid network
```

Comment:
The "implementation-specific diagnostic information" conventionally includes at least the text of the assertion condition, and the filename and the line number. In fact this is required in POSIX: http://pubs.opengroup.org/onlinepubs/9699919799/functions/assert.html